.husky/addTranslations.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+yarn build:extension:translations
+git diff --quiet || (echo "Adding translations commit. Please retry git push" && git add . && git commit -m "Added translations" && exit 1)
```

Comment:
on push, this runs the translate job, determines if any translations need to be added, and then aborts the push if translations are found. It will automatically commit and stage the translations, so all you need to do is `git push` again (or `git push --no-verify` if you don't want to run the whole translate job again and you've made no other changes)